packages/ffe-buttons-react/src/ActionButton.js

```diff
@@ -35,6 +35,8 @@ ActionButton.propTypes = {
     disabled: bool,
     /** The rendered element, like an `<a />` or `<Link />` */
     element: oneOfType([func, string]),
+    /** Applies the ghost modifier if true. */
+    ghost: bool,
```

Comment:
This change isn't really related to `docs`, so I think it should either be skipped or done in its own commit.

Reply:
It kind of is related though, since it changes the order this prop shows up in the docs.

Reply:
That's a stretch, but I'll allow it :wink: